src/openforms/authentication/contrib/digid_eherkenning_oidc/plugin.py

```diff
@@ -152,14 +152,17 @@ class DigiDMachtigenOIDCAuthentication(OIDCAuthentication):
 
     def add_claims_to_sessions_if_not_cosigning(self, claim, request):
         # set the session auth key only if we're not co-signing
-        if claim and CO_SIGN_PARAMETER not in request.GET:
-            config = OpenIDConnectDigiDMachtigenConfig.get_solo()
-            request.session[FORM_AUTH_SESSION_KEY] = {
-                "plugin": self.identifier,
-                "attribute": self.provides_auth,
-                "value": claim[config.vertegenwoordigde_claim_name],
-                "machtigen": request.session[DIGID_MACHTIGEN_OIDC_AUTH_SESSION_KEY],
-            }
+        if not claim or CO_SIGN_PARAMETER in request.GET:
+            return
+
+        config = self.config_class.get_solo()
+        machtigen_data = request.session[DIGID_MACHTIGEN_OIDC_AUTH_SESSION_KEY]
```

Comment:
this could be

```suggestion
        machtigen_data = request.session[self.session_key]
```
to be consistent with the `self.config_class` above?